pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1485,4 +1485,13 @@ self: super: {
     sha256 = "097wqn8hxsr50b9mhndg5pjim5jma2ym4ylpibakmmb5m98n17zp";
   });
 
+  # hackage version does not support newer servant api
+  hoggl = overrideSrc super.hoggl {
+    src = pkgs.fetchFromGitHub {
```

Comment:
Overriding `src` with a specific github revision is pretty ugly. The danger we face doing this is that we'll keep shipping the unofficial version even after a new release has been made on Hackage. It would be much better to (a) lobby upstream to release a new version or (b) apply a *patch* to the existing version so that the build will fail if a new version comes out that has that patch applied already. Then we'll notice that the patch can be dropped.

Reply:
Oh, yes, thanks! I totally forgot about the `appendPatch`/`fetchpatch` option. I'm trying to do (a) in the mean time https://github.com/markus1189/hoggl/issues/12